DSCPullServerSetup/PublishModulesAndMofsToPullServer.psm1

```diff
@@ -1,34 +1,11 @@
-<#
```

Comment:
Thank you for updating the function help comment! But can you please do not move the function help comment into the function? It's actually part of our [DSC Resource Kit style guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#functions-have-comment-based-help) that it comes before the function.

This comment goes for the rest of the new function help comments here as well.

Otherwise this PR looks great 😄